### PR TITLE
Translate 2016 Fukuoka Ruby Award (zh_cn)

### PR DESCRIPTION
**Pull request by Zane5**
Related #1194

screen shot 2015-10-29 at 17 28 55

### CODE REVIEW

File: zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: news_post
+title: "2016 日本福冈 Ruby 有奖竞赛 - 松本行弘评判"
+author: "Fukuoka Ruby"
+translator: Zane5
+date: 2015-10-13 08:00:00 +0000
+lang: zh_cn
+---
+
+亲爱的 Ruby 爱好者,
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Please use Chinese comma.

File: zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+项目必须开发于或完成于过去的 12 个月内。请访如下链接获取详情：
+
+[http://www.digitalfukuoka.jp/events/84](http://www.digitalfukuoka.jp/events/84)
+或
+[http://myfukuoka.com/events/2016-fukuoka-ruby-award-guidelines-for-applicants/](http://myfukuoka.com/events/2016-fukuoka-ruby-award-guidelines-for-applicants/)
+
+[http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc](http://www.digitalfukuoka.jp/uploads/event_detail/file/210/RubyAward_ApplicationForm_2016.doc)
+
+请发送申请表格到 award@f-ruby.com
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2015-10-13-fukuoka-ruby-award-2016.md]: Missing period `。`?

**Comment by JuanitoFatas** [comment]
Other than my comments, LGTM. :+1: 

P.S. [Commit messages could be improved](https://github.com/ruby/www.ruby-lang.org/wiki#commits-and-commit-messages) :blush:

**Comment by hlcfan** [comment]
:boom: Thank you!

**Comment by stomar** [comment]
Yep, like `Translate 2015-10-13-fukuoka-ruby-award-2016 (zh_cn)` or e.g. `Translate 2016 Fukuoka Ruby Award (zh_cn)`, not more than 50 characters if possible.

Please amend your commit for these tiny changes (`git commit --amend`) and also fix the commit message.

**Comment by Zane5** [comment]
I have fixed Chinese comma and changed the commit with `git commit --amend`
but I can not push it.
need some help.

```
± % git status
On branch zh_cn/fukuoka-ruby-award-2016
nothing to commit, working directory clean
```

```
git push zane5 zh_cn/fukuoka-ruby-award-2016                                     

To git@github.com:zane5/www.ruby-lang.org.git
 ! [rejected]        zh_cn/fukuoka-ruby-award-2016 -> zh_cn/fukuoka-ruby-award-2016 (non-fast-forward)
error: failed to push some refs to 'git@github.com:zane5/www.ruby-lang.org.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

```
± %  git pull origin master                                                          !2371
Enter passphrase for key '/home/vagrant/.ssh/id_rsa':
From github.com:ruby/www.ruby-lang.org
 * branch            master     -> FETCH_HEAD
Already up-to-date.
```

**Comment by JuanitoFatas** [comment]
You need to force push this branch to update commit message: `git push zane5 zh_cn/fukuoka-ruby-award-2016 -f`, [see docs on --force option](https://www.kernel.org/pub/software/scm/git/docs/git-push.html).

**Comment by Zane5** [comment]
@JuanitoFatas thanks :)

**Comment by stomar** [comment]
:+1:

**Comment by JuanitoFatas** [comment]
:+1:

**Pull request merged**